profiles/adb.jsonld

```diff
@@ -0,0 +1,231 @@
+  "prefLabel": {
+  	"en": "Actionable Data Book (ADB) Profile"
+  },
+  "definition": {
+    "en": "The xAPI ABD profile is used to identify interactions between an actor and an eBook activity based on the IEEE ADB project."
+  },
+  "seeAlso": "https://standards.ieee.org/develop/indconn/adb/",
+  "author": {
+    "type": "Group",
+    "name": "IEEE Industry Connections Actionable Data Book (ADB) Team"
+  },
+  "versions": [
+    {
+      "id": "https://w3id.org/xapi/adb/v1",
+      "generatedAtTime": "2017-06-30T8:26:00Z",
+      "wasRevisionOf": ["https://w3id.org/xapi/adb"]
+    }
+  ],
+  "concepts": [
+    {
+      "id": "https://w3id.org/xapi/adb/verbs/annotated",
+      "inScheme": "https://w3id.org/xapi/adb",
+      "type": "Verb",
+      "definition": {
+        "en": "Provide notes or symbols for further explanation or thoughts while reading."
+      },
+      "narrower": [ {
+          "id": "https://w3id.org/xapi/adb/verbs/highlighted"
+      } ],
+      "relatedMatch": [ {
+          "id": "https://w3id.org/xapi/acrossx/verbs/annotated"
+      } ],
+      	"prefLabel": {
+        "en": "annotated"
+      }
+    },
+    {
+      "id": "https://w3id.org/xapi/adb/verbs/arrived",
+      "inScheme": "https://w3id.org/xapi/adb",
+      "type": "Verb",
+      "broadMatch": {
+        "id": "http://adlnet.gov/expapi/verbs/progressed"
+      },
+      "definition": {
+        "en": "Indicates the actor arrived a specific location by movement or progress. "
+      },
+      "scopeNote": {
+        "en": "To arrive a specific scene within a story or book experience."
+      },
+        "prefLabel": {
+        "en": "arrived"
+      }
+    },
+    {
+      "id": "https://w3id.org/xapi/adb/verbs/attended",
+      "inScheme": "https://w3id.org/xapi/adb",
+      "type": "Verb",
+      "definition": {
+        "en": "Indicates the actor was present at a virtual or physical event or activity. "
+      },
+        "prefLabel": {
+        "en": "attended"
+      }
+    },
+    {
+      "id": "https://w3id.org/xapi/adb/verbs/bookmarked",
+      "inScheme": "https://w3id.org/xapi/adb",
+      "type": "Verb",
+      "definition": {
+        "en": "Persisting the current location (page) where the reader stopped the ebook activity."
+      },
+        "prefLabel": {
+        "en": "bookmarked"
+      },
+      "related": [ {
+        "id": "https://w3id.org/xapi/adb/verbs/annotated"
+      } ]
+    },
+    {
+      "id": "https://w3id.org/xapi/adb/verbs/coached",
+      "inScheme": "https://w3id.org/xapi/adb",
+      "type": "Verb",
+      "definition": {
+        "en": "To teach or lead another person through an activity. Coached is a direct response to the Requested verb. The receiver of a Requested statement will periodically respond by sending a Coached statement that clears out the coach's queue."
+      },
+        "prefLabel": {
+        "en": "coached"
+      },
+      "related": [ {
```

Comment:
These are not related. Related only means the meaning is close, it is not a general "there is some sort of relationship". That one is a response to the other, a causal relationship, does not mean their meanings are close -- it means their meanings are very different.